flowzone.yml

```diff
@@ -704,10 +709,11 @@ jobs:
           fi
 
       # Check if we have atleast a README to build a website from
+
       - name: Check for README for building a website
         id: has_readme
         run: |
-          if test -e "README.md"
+          if test -f "README.md"
```

Comment:
any reason to require a file? might be a symlink?

Reply:
This change might have snuck in. Reverting.